examples/experimental/omniglot_protonet_pytorch/README.md

```diff
@@ -0,0 +1,24 @@
+## Prototypical Networks on Omniglot
+This example implements a meta-learning method called prototypical networks (ProtoNets) for few-shot learning on the Omniglot dataset.  ProtoNets often outperform MAML on few-shot learning problems and is one of the most popular methods for meta-learning.
```

Comment:
This example implements a meta-learning method... -> This example implements a popular meta-learning method. 

remove: "and is one of the most popular methods ...."